src/awkward/_connect/pyarrow_table_conv.py

```diff
@@ -0,0 +1,250 @@
+from __future__ import annotations
+
+import json
+
+from .pyarrow import (
+    AwkwardArrowArray,
+    AwkwardArrowType,
+    import_pyarrow,
+    to_awkwardarrow_storage_types,
+)
+
+pyarrow = import_pyarrow("pyarrow_table_conv")
+AWKWARD_INFO_KEY = b"ak_extn_array_info"  # metadata field in Table schema
+
+
+def convert_awkward_arrow_table_to_native(aatable: pyarrow.Table) -> pyarrow.Table:
+    """
+    aatable: A pyarrow Table created with extensionarray=True
+    returns: A pyarrow Table without extensionsarrays, but
+      with 'awkward_info' in the schema's metadata that can be used to
```

Comment:
Do you mean

```suggestion
      with 'awkward_array_metadata' in the schema's metadata that can be used to
```

Is this documentation referring to the value of `AWKWARD_INFO_KEY`?

Reply:
Oh, yes it is. I shot myself in the proverbial foot.
Can I put ``f""" ... `{AWKWARD_INFO_KEY}` ... """`` in the docstring?
Looks like that should work. Less footgun.

Reply:
It did not work, and Ruff caught the would-be-error thankfully. I just corrected it manually.